ui/lib/ldap/addon/components/page/library/details/accounts.hbs

```diff
@@ -4,14 +4,12 @@
       <div class="is-flex-between">
         <h3 class="is-size-5 has-text-weight-semibold">All accounts</h3>
         {{#if @library.canCheckOut}}
-          <button
-            type="button"
-            class="button is-link"
+          <Hds::Button
+            @text="Check-out"
+            @color="secondary"
```

Comment:
Seeing `is-link` looks like this would should also be tertiary if we add an icon

Reply:
Good call I talked to design about an icon and updated it!

Reply:
Nice! so the decision was to keep this `@secondary` or include a custom icon? Adding to my notes about swapping 😄

Reply:
Tertiary with an icon. Not sure if this will be the solution for every case. I think it will either have to be tertiary + icon or text + secondary.

Reply:
Yep - I'm adding a note that if it opens a modal - to consult design